test/Interop/Cxx/value-witness-table/non-trivial-implicit-destructor-irgen.swift

```diff
@@ -0,0 +1,18 @@
+// RUN: %target-swift-frontend -enable-cxx-interop -I %S/Inputs %s -emit-ir | %FileCheck %s
```

Comment:
I don't think this test belongs in the value-witness-table directory. Could you move it to the class directory?